arch/arc/include/swap_macros.h

```diff
@@ -365,6 +365,18 @@
 #endif
 .endm
 
+/* macro to push aux reg through reg */
+.macro pushax reg aux
```

Comment:
May we define macros in ALL CAPITALS?
Otherwise people will get confused thinking we do have real instructions which do AUX regs POP/PUSH.